internal/kubernetes/pvc.go

```diff
@@ -63,6 +64,34 @@ func ListAllPVCWithStorageclass(client *k8s.Clientset, scName string) (*[]corev1
 	return pl, nil
 }
 
+func ListSinglePVCWithStorageclass(client *k8s.Clientset, scName, pvcName, pvcNamespace string) (*[]corev1.PersistentVolumeClaim, error) {
```

Comment:
Shouldn't need the storage class name if we're getting a pvc directly. Hopefully you can just look up the storage class referenced by the PVC so that it isn't needed as a command line param either.
```suggestion
func ListSinglePVCWithStorageclass(client *k8s.Clientset, pvcName, pvcNamespace string) (*[]corev1.PersistentVolumeClaim, error) {
```

Reply:
changed, JFYI I have not tested the latest changes as waiting for Yug's PR to get merged.